avm/res/desktop-virtualization/host-pool/tests/e2e/max/main.test.bicep

```
@@ -0,0 +1,156 @@
+targetScope = 'subscription'
+
+metadata name = 'Using all available parameters'
+metadata description = 'This instance deploys the module with all available parameters for testing purposes.'
```

```suggestion
metadata name = 'Using large parameter set'
metadata description = 'This instance deploys the module with most of its features enabled.'
```

'All' is a strong statement because you can usually not actually test all properties at ones. We hence usually opt for a less absolute statement ;)